CONTRIBUTING.md

```diff
@@ -55,35 +55,35 @@ Things that appear to be broken or are not working as intended.
 
 An enhancement to an existing feature.
 
-### [`external dependency` ![Issues tagged with 'external dependency'](https://img.shields.io/github/issues-raw/nasa-gibs/worldview/external%20dependency.svg)](https://github.com/nasa-gibs/worldview/issues?q=is%3Aopen+is%3Aissue+label%3Aexternal%20dependency)
+### [`external dependency` ![Issues tagged with 'external dependency'](https://img.shields.io/github/issues-raw/nasa-gibs/worldview/external%20dependency.svg)](https://github.com/nasa-gibs/worldview/issues?q=is%3Aopen%20is%3Aissue%20label%3A"external%20dependency")
 
 Issues that are waiting on something out of our control.
 
-### [`help wanted` ![Issues tagged with 'help wanted'](https://img.shields.io/github/issues-raw/nasa-gibs/worldview/help%20wanted.svg)](https://github.com/nasa-gibs/worldview/issues?q=is%3Aopen+is%3Aissue+label%3Ahelp%20wanted)
+### [`help wanted` ![Issues tagged with 'help wanted'](https://img.shields.io/github/issues-raw/nasa-gibs/worldview/help%20wanted.svg)](https://github.com/nasa-gibs/worldview/issues?q=is%3Aopen+is%3Aissue+label%3A"help%20wanted")
```

Comment:
Looking good. Could you also update the svg tags to include quotes?
i.e. https://img.shields.io/github/issues-raw/nasa-gibs/worldview/"help%20wanted".svg

Reply:
Ah, ok

Reply:
Make sure you encode those quotes with `%22` :)

Reply:
Good catch @localjo, 
the svg's would be: https://img.shields.io/github/issues-raw/nasa-gibs/worldview/%22help%20wanted%22.svg

and the links would be https://github.com/nasa-gibs/worldview/issues?q=is%3Aopen+is%3Aissue+label%3A%22help%20wanted%22

Reply:
Shure, fixed